hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/volume/TestVolumeSetDiskChecks.java

```diff
@@ -141,6 +143,9 @@ HddsVolumeChecker getVolumeChecker(Configuration configuration)
         return new DummyChecker(configuration, new Timer(), numVolumes);
       }
     };
+
+    assertEquals(volumeSet.getFailedVolumesList().size(), numVolumes);
```

Comment:
This is updated like this due to code reordering. As we are calling checkAllVolumes() when only volumeMap size is not zero.